sdk/python/flet/tooltip.py

```diff
@@ -0,0 +1,254 @@
+        return children
+
+    # enable_feedback
+    @property
+    def enable_feedback(self) -> Optional[bool]:
+        return self._get_attr("enableFeedback", data_type="bool", def_value=False)
+
+    @enable_feedback.setter
+    @beartype
+    def enable_feedback(self, value: Optional[bool]):
+        self._set_attr("enableFeedback", value)
+
+    # margin
+    @property
+    def margin(self) -> MarginValue:
+        return self.__margin
+
+    @margin.setter
+    @beartype
+    def margin(self, value: MarginValue):
+        self.__margin = value
+
+    # padding
+    @property
+    def padding(self) -> PaddingValue:
+        return self.__padding
+
+    @padding.setter
+    @beartype
+    def padding(self, value: PaddingValue):
+        self.__padding = value
+
+    # bgcolor
+    @property
+    def bgcolor(self):
+        return self._get_attr("bgColor")
+
+    @bgcolor.setter
+    def bgcolor(self, value):
+        self._set_attr("bgColor", value)
+
+    # gradient
+    @property
+    def gradient(self) -> Optional[Gradient]:
+        return self.__gradient
+
+    @gradient.setter
+    @beartype
+    def gradient(self, value: Optional[Gradient]):
+        self.__gradient = value
+
+    # border
+    @property
+    def border(self) -> Optional[Border]:
+        return self.__border
+
+    @border.setter
+    @beartype
+    def border(self, value: Optional[Border]):
+        self.__border = value
+
+    # border_radius
+    @property
+    def border_radius(self) -> BorderRadiusValue:
+        return self.__border_radius
+
+    @border_radius.setter
+    @beartype
+    def border_radius(self, value: BorderRadiusValue):
+        self.__border_radius = value
+
+    # shape
+    @property
+    def shape(self):
+        return self._get_attr("shape")
+
+    @shape.setter
+    @beartype
+    def shape(self, value: Optional[BoxShape]):
+        self._set_attr("shape", value.value if value is not None else None)
+
+    # message
+    @property
+    def message(self) -> Optional[str]:
+        return self._get_attr("message")
+
+    @message.setter
+    @beartype
+    def message(self, value: Optional[str]):
+        self._set_attr("message", value)
+
+    # text_align
+    @property
+    def text_align(self) -> TextAlign:
+        return self._get_attr("textAlign")
+
+    @text_align.setter
+    @beartype
+    def text_align(self, value: TextAlign):
+        self._set_attr("textAlign", value)
+
+    # text_style
+    @property
+    def text_style(self):
+        return self.__text_style
+
+    @text_style.setter
+    @beartype
+    def text_style(self, value: Optional[TextStyle]):
+        self.__text_style = value
+
+    # prefere_below
+    @property
+    def prefere_below(self) -> Optional[bool]:
+        return self._get_attr("prefereBelow", data_type="bool", def_value=False)
+
+    @prefere_below.setter
+    @beartype
+    def prefere_below(self, value: Optional[bool]):
+        self._set_attr("prefereBelow", value)
```

Comment:
There is a little issue over here. I had to come right to this commit to see how the attribute was named in the `tooltip.dart`.
The little issue is the a simple grammar mistake: `prefer` and not `prefere`. (notice the `e` at the end)
Will make a PR for this right now.